libexec/goenv-install

```diff
@@ -41,11 +41,12 @@ if [ "$platform" = "darwin" ]; then
       extra="-osx10.6"
     fi
   fi
+  # 1.4.3 later extra none
+  if [ "$version" \> "1.4.2" ]; then
```

Comment:
Can this use the `vercomp` function? Feel free to move the definition of `vercomp` up.